### ISSUE TITLE
Error when deploying to Elastic Beanstalk

### ISSUE THREAD

**Issue by DaveMiscampbell** [appveyor/ci#2521]
Hi there,

When using Elastic Beanstalk as an Environment - I get the following build log:

```...environment '...' found and available for redeployment.
...starting redeployment to AWS Elastic Beanstalk environment '...'
...making sure upload bucket '...' exists
..a bucket with name '...' already exists and will be used for upload
....uploading application deployment package to Amazon S3
......uploading from file ...zip, size 40439699 bytes
Base stream of PartialWrapperStream must be seekable```

And the deployment fails.

Is there something I'm doing incorrectly? The artifact I'm trying to deploy is listed as type "Elastic Beanstalk package".

Thanks for your help

**Comment by IlyaFinkelshteyn** [comment]
Can you please share your build configuration in YAML format and your Environment settings screenshot? Sure please delete all secrets from the screenshot and ensure secrets in YAML are encrypted (or also delete them). If you are not comfortable sharing here, please send YAML and screenshot to `team at appveyor.com`. Thanks!

**Comment by IlyaFinkelshteyn** [comment]
@DaveMiscampbell do you still have this issue? We just got another case like this and do not have a repro ourselves... Any input would be helpful.

**Comment by parshikovalexey** [comment]
@IlyaFinkelshteyn, attach screen from AWS EB
![eb](https://user-images.githubusercontent.com/11713323/49855334-82668c00-fe1f-11e8-92e1-997bdc7c6397.png)

**Comment by IlyaFinkelshteyn** [comment]
@parshikovalexey can you please try adding `image: Visual Studio 2017` to the YAML and see if this behaves differently after that?

**Comment by parshikovalexey** [comment]
It's not helped. I update project settings. Make a new build and then try to deploy it. And I have same error.

**Comment by IlyaFinkelshteyn** [comment]
@parshikovalexey is this project open source?

**Comment by parshikovalexey** [comment]
@IlyaFinkelshteyn, it's private project, which placed in the bitbucket. Do you think we have some problems with out VS solution? We have .ebextensions in the our project. Could it create any problems?

**Comment by IlyaFinkelshteyn** [comment]
I do not think so. Can you please share a link to the build? You can send it to support@appveyor.com if you do not feel comfortable sharing it here.

**Comment by parshikovalexey** [comment]
I sent link to email above as you asked, But didn't get any answers. Could you provide some ETA for your investigation of this problem?

**Comment by IlyaFinkelshteyn** [comment]
We checked the log and did not thing anything wrong with it. Can you share that build artifact for experiments?

**Comment by parshikovalexey** [comment]
Any news with such problem?
